p2p/src/authenticated/actors/tracker/actor.rs

```diff
@@ -157,6 +156,8 @@ pub struct Actor<E: Spawner + Rng + GClock, C: Scheme> {
 }
 
 impl<E: Spawner + Rng + Clock + GClock, C: Scheme> Actor<E, C> {
+    const NAMESPACE_SUFFIX_IP: &[u8] = b"_IP";
```

Comment:
nit: I suggest we still define this at the top of the file (like it was in the last review).